deploy/kubernetes/console/README.md

```diff
@@ -75,7 +75,7 @@ The following table lists the configurable parameters of the Stratos Helm chart
 |console.templatesConfigMapName|Name of config map that provides the template files for user invitation emails||
 |console.userInviteSubject|Email subject of the user invitation message||
 |console.techPreview|Enable/disable Tech Preview features|false|
-|console.apiKeysEnabled|Enable/disable API key-based access to Stratos API (disable, admin_only, all_users)|admin_only|
+|console.apiKeysEnabled|Enable/disable API key-based access to Stratos API (disabled, admin_only, all_users)|admin_only|
```

Comment:
@richard-cox thanks for catching that! :+1:

Reply:
It was an easy catch after jetstream failed to start! That validation saved a good amount of time